names.py

```diff
@@ -0,0 +1,164 @@
+    93: 'Spectrum', 94: 'Ectoplasma', 95: 'Onix', 96: 'Soporifik',
+    97: 'Hypnomade', 98: 'Krabby', 99: 'Krabboss', 100: 'Voltorbe',
+    101: 'Électrode', 102: 'Nœunœuf', 103: 'Noadkoko', 104: 'Osselait',
+    105: 'Ossatueur', 106: 'Kicklee', 107: 'Tygnon', 108: 'Excelangue',
+    109: 'Smogo', 110: 'Smogogo', 111: 'Rhinocorne', 112: 'Rhinoféros',
+    113: 'Leveinard', 114: 'Saquedeneu', 115: 'Kangourex', 116: 'Hypotrempe',
+    117: 'Hypocéan', 118: 'Poissirène', 119: 'Poissoroy', 120: 'Stari',
+    121: 'Staross', 122: 'M. Mime', 123: 'Insécateur', 124: 'Lippoutou',
+    125: 'Élektek', 126: 'Magmar', 127: 'Scarabrute', 128: 'Tauros',
+    129: 'Magicarpe', 130: 'Léviator', 131: 'Lokhlass', 132: 'Métamorph',
+    133: 'Évoli', 134: 'Aquali', 135: 'Voltali', 136: 'Pyroli',
+    137: 'Porygon', 138: 'Amonita', 139: 'Amonistar', 140: 'Kabuto',
+    141: 'Kabutops', 142: 'Ptéra', 143: 'Ronflex', 144: 'Artikodin',
+    145: 'Électhor', 146: 'Sulfura', 147: 'Minidraco', 148: 'Draco',
+    149: 'Dracolosse', 150: 'Mewtwo', 151: 'Mew'
+}
+
+CHINESE_NAMES = {
+    1: '妙蛙种子', 2: '妙蛙草', 3: '妙蛙花', 4: '小火龙', 5: '火恐龙',
+    6: '喷火龙', 7: '杰尼龟', 8: '卡咪龟', 9: '水箭龟', 10: '绿毛虫',
+    11: '铁甲蛹', 12: '巴大蝶', 13: '独角虫', 14: '铁壳蛹', 15: '大针蜂',
+    16: '波波', 17: '比比鸟', 18: '大比鸟', 19: '小拉达', 20: '拉达',
+    21: '烈雀', 22: '大嘴雀', 23: '阿柏蛇', 24: '阿柏怪', 25: '皮卡丘',
+    26: '雷丘', 27: '穿山鼠', 28: '穿山王', 29: '尼多兰', 30: '尼多娜',
+    31: '尼多后', 32: '尼多朗', 33: '尼多力诺', 34: '尼多王', 35: '皮皮',
+    36: '皮可西', 37: '六尾', 38: '九尾', 39: '胖丁', 40: '胖可丁',
+    41: '超音蝠', 42: '大嘴蝠', 43: '走路草', 44: '臭臭花', 45: '霸王花',
+    46: '派拉斯', 47: '派拉斯特', 48: '毛球', 49: '摩鲁蛾', 50: '地鼠',
+    51: '三地鼠', 52: '喵喵', 53: '猫老大', 54: '可达鸭', 55: '哥达鸭',
+    56: '猴怪', 57: '火暴猴', 58: '卡蒂狗', 59: '风速狗', 60: '蚊香蝌蚪',
+    61: '蚊香君', 62: '蚊香泳士', 63: '凯西', 64: '勇基拉', 65: '胡地',
+    66: '腕力', 67: '豪力', 68: '怪力', 69: '喇叭芽', 70: '口呆花',
+    71: '大食花', 72: '玛瑙水母', 73: '毒刺水母', 74: '小拳石', 75: '隆隆石',
+    76: '隆隆岩', 77: '小火马', 78: '烈焰马', 79: '呆呆兽', 80: '呆壳兽',
+    81: '小磁怪', 82: '三合一磁怪', 83: '大葱鸭', 84: '嘟嘟', 85: '嘟嘟利',
+    86: '小海狮', 87: '白海狮', 88: '臭泥', 89: '臭臭泥', 90: '大舌贝',
+    91: '刺甲贝', 92: '鬼斯', 93: '鬼斯通', 94: '耿鬼', 95: '大岩蛇',
+    96: '催眠貘', 97: '引梦貘人', 98: '大钳蟹', 99: '巨钳蟹', 100: '霹雳电球',
+    101: '顽皮雷弹', 102: '蛋蛋', 103: '椰蛋树', 104: '卡拉卡拉',
+    105: '嘎啦嘎啦', 106: '飞腿郎', 107: '快拳郎', 108: '大舌头',
+    109: '瓦斯弹', 110: '双弹瓦斯', 111: '独角犀牛', 112: '钻角犀兽',
+    113: '吉利蛋', 114: '蔓藤怪', 115: '袋兽', 116: '墨海马', 117: '海刺龙',
+    118: '角金鱼', 119: '金鱼王', 120: '海星星', 121: '宝石海星',
+    122: '魔墙人偶', 123: '飞天螳螂', 124: '迷唇姐', 125: '电击兽',
+    126: '鸭嘴火兽', 127: '凯罗斯', 128: '肯泰罗', 129: '鲤鱼王',
+    130: '暴鲤龙', 131: '拉普拉斯', 132: '百变怪', 133: '伊布', 134: '水伊布',
+    135: '雷伊布', 136: '火伊布', 137: '多边兽', 138: '菊石兽',
+    139: '多刺菊石兽', 140: '化石盔', 141: '镰刀盔', 142: '化石翼龙',
+    143: '卡比兽', 144: '急冻鸟', 145: '闪电鸟', 146: '火焰鸟', 147: '迷你龙',
+    148: '哈克龙', 149: '快龙', 150: '超梦', 151: '梦幻'
+}
+
+try:
```

Comment:
More concise, and I think more Pythonic:

```py
POKEMON_NAMES = {
    'German': GERMAN_NAMES,
    'French': FRENCH_NAMES,
    'Chinese': CHINESE_NAMES,
}.get(config.LANGUAGE, ENGLISH_NAMES)
```

Also, could you use 2-letter (or 3-letter) codes instead of English language name?